pkg/catalogsourceconfig/reconciler.go

```diff
@@ -0,0 +1,36 @@
+package catalogsourceconfig
+
+import (
+	"context"
+
+	"github.com/operator-framework/operator-marketplace/pkg/apis/marketplace/v1alpha1"
+)
+
+// Reconciler is the interface that wraps the Reconcile method.
+//
+// Reconcile reconciles a particular phase of the CatalogSourceConfig object
+// and returns the next desired phase.
+//
+// in represents the original CatalogSourceConfig object received from the sdk
+// and before reconciliation has started.
+//
+// out represents the CatalogSourceConfig object after reconciliation has
+// completed and could be different from the original. The CatalogSourceConfig
```

Comment:
I think that `and can be different` is better, but I'm not sure.

Reply:
`can` implies it is definitely different which is not the case here.